Library/Formula/squirrel-sql.rb

```diff
@@ -0,0 +1,43 @@
+class SquirrelSql < Formula
+  desc "GUI for JDBC-compliant databases"
+  homepage "http://www.squirrelsql.org/"
+  head "git://git.code.sf.net/p/squirrel-sql/git"
+
+  depends_on "ant" => :build
+  depends_on :java
+
+  def install
+    cd "sql12" do
+      system "ant"
+      cd buildpath/"sql12/output/plainZip" do 
+        zipfile = Dir.glob("squirrelsql-snapshot-*_*-standard.zip")[0]
+        system "unzip", zipfile
+        unzipped = zipfile[0 ... -4] # trim off the ".zip"
+        cd unzipped do
+          # install the libraries
+          lib.install Dir["lib/*.jar"]
```

Comment:
This is the easiest way to get it to work for now; at some point we'll want to unbundle these libraries though.